models/DataObject/AbstractObject.php

```diff
@@ -115,6 +116,17 @@ abstract class AbstractObject extends Model\Element\AbstractElement
      */
     protected ?string $childrenSortOrder = null;
 
+    /**
+     * @internal
+     */
+    protected ?string $classId = null;
```

Comment:
I'm sorry, but we have to revert the last commit. In the concrete classes it is also declared without native types and a syntax error occurs.

```suggestion
    /**
     * @internal
     *
     * @var string|null
     */
    protected $classId = null;
```